etl-job/fhir_import_export.py

```diff
@@ -238,29 +211,10 @@ def _load_all(study: str,
             meta_flat_delete(project_id=f"{program}-{project}", index=index)
 
         for index, generator in index_generator_dict.items():
-            load_flat(project_id=project_id, index=index,
-                    generator=generator(),
-                    limit=None, elastic_url=DEFAULT_ELASTIC,
-                    output_path=None)
-
-    # when generating graph with jsonschemagraph
-    except subprocess.CalledProcessError as exception:
-        # save and print any useful logs
-        tb = traceback.print_tb(exception.__traceback__)
-        for title, log in [("stdout", exception.stdout), ("traceback", tb), ("ERROR", exception.stderr)]:
-            if not log:
-                continue
-            
-            message = f"{title.upper()}: {log}"
-            output['logs'].append(message)
-            _write_output_to_client(output)
-            print(message)
-
-        # print final error before raising
-        final_error = f"ERROR: Unable to generate valid jsonschema graph from {file_path} to {extraction_path} for project ID {project_id}"
-        output['logs'].append(final_error)
-        _write_output_to_client(output)
-        raise
+            load_flat(project_id=f"{program}-{project}", index=index,
+                      generator=generator(),
+                      limit=None, elastic_url=DEFAULT_ELASTIC,
+                      output_path=None)
```

Comment:
Fine with exiting the job after grip load if there is >0 validation errors by default, but the --ignore-validation g3t option that currently ignores client side validation needs to be passed as an arg to the fhir import job  and used in the fhir import job so that validation errors can also be ignored there too if the flag is specified.

Reply:
Responding to your DM'ed comment:
> I’m fine with exiting the job after grip load if there is >0 validation errors by default, but the --ignore-validation g3t option that currently ignores client side validaiton needs to be passed as an arg to the fhir import job  and used in the fhir import job so that validation errors can also be ignored there too if the flag is specified.

hmmm yeah that's true, so what `g3t push --skip_validate` would do in that case is continue to load Elastic data so that at least a partial graph can be uploaded? That makes sense to me, though I think the `skip_validate` was primarily to address the "project ID doesn't match UUIDs but still wanna upload entire graph" use case (see [here](https://github.com/ACED-IDP/gen3_util/pull/104)). Was there another use case that you felt needed to be covered?

Otherwise, I'm okay to just keep the scope as exiting the job on validation error and then we can make the "validate as an arg for users loading a partial graph" a separate ticket and user story.

Lmk if that makes sense.
@bwalsh any opinions on this?

Reply:
Oh and speaking of tickets, is there a Jira ticket to track this work?

Reply:
End user shouldn't be able to skip server validation, but realistically front end developer developing components for specific datasets should. 

Front-end developer shouldn't have to deal with 50 validation errors in a dataset of 50,000 rows where those 50 invalid rows aren't part of the new feature that needs to get added and those 50 validation errors will get solved in the future by another developer.

There should be a balance here. Like an Arborist permission that can be given to a superuser to be able to skip validation server side for rapid local development.

Reply:
> I’m fine with exiting the job after grip load if there is >0 validation errors by default 

Agreed

> g3t push --skip_validate ...

QW is correct in that the flag was implemented to "primarily to address the "project ID doesn't match UUIDs but still wanna upload entire graph".  In other words to support re-using data for testing in different program-projects.

In my opinion, we should:
* return error immediately if there is a validation or other error loading grip, do not load dataframes
* make the --skip_validate option a hidden flag - it should only be used under controlled tests/devops

>  speaking of tickets 

* This discussion seems to be part of an existing ticket.  https://jirabp.ohsu.edu/browse/TKEDTE-277
* On consensus, I would just update that ticket with these details

Reply:
Consensus reached. See https://github.com/ACED-IDP/aced_etl_pod/pull/43/commits/d00414e3ab2421c783f85f607ee21314a293355f